tests/metrics/test_metric.py

```diff
@@ -13,7 +13,7 @@
 TABLE = "my_table"
 COLUMN = "my_column"
 
-FULLY_QUALIFIED_METRIC_NAME = "column_values.above"
```

Comment:
This is meant to look like [this](https://github.com/great-expectations/great_expectations/blob/6c3f8de396495746a12d655aecc31c62d40c75d1/great_expectations/expectations/metrics/column_map_metrics/column_values_between.py#L23)

Reply:
same point as above, the fully qualified metric name is meant for consumption by core internals

Reply:
updated in